mmdeploy/utils/env.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import importlib
+
+from mmdeploy.utils import Codebase
+
+
+def get_library_version(lib):
+    """Try to get the version of a library if it has been installed.
+
+    Args:
+        lib (str): The name of library.
+
+    Returns:
+        None | str: If the library has been installed, return version.
+    """
+    try:
+        lib = importlib.import_module(lib)
+    except ImportError:
```

Comment:
load library might throw other exception.
Such as mmocr will assert the mmcv verison. which will thown an assert error.